LoRaEngine/modules/LoRaWanNetworkSrvModule/LoraTools/Regions/Region.cs

```diff
@@ -267,6 +267,46 @@ public string GetDownstreamRX2DataRate(string devEUI, string nwkSrvRx2Dr, ushort
             return defaultDatr;
         }
 
+        /// <summary>
+        /// Get downstream RX2 data rate.
+        /// </summary>
+        /// <param name="devEUI">The device id.</param>
+        /// <param name="nwkSrvRx2Dr">The network server rx2 datarate.</param>
+        /// <param name="rx2DrFromTwins">RX2 datarate value from twins.</param>
+        /// <returns>The RX2 data rate.</returns>
+        public ushort GetDownstreamRX2DataRate(string devEUI, ushort? nwkSrvRx2Dr, ushort? rx2DrFromTwins, ILogger logger, DeviceJoinInfo deviceJoinInfo = null)
```

Comment:
This got deleted accidentally in PR https://github.com/Azure/iotedge-lorawan-starterkit/commit/e2b1b7bac49c7cf9a474c50ddef6158c150d05d7#diff-2890d1a0249a453790265b3dfd66d4661eb35fcc79183e05ae8ec16bd78bf0e6